src/components/lab/NumberInput/NumberInput.scss

```
@@ -0,0 +1,44 @@
+@use '../../variables';
+
+$block: '.#{variables.$ns}number-input';
+
+#{$block} {
+    &_size {
+        &_s {
+            --_--textinput-end-padding: 1px;
+        }
+
+        &_m {
+            --_--textinput-end-padding: 1px;
+        }
+
+        &_l {
+            --_--textinput-end-padding: 3px;
+        }
+
+        &_xl {
+            --_--textinput-end-padding: 3px;
+        }
+    }
+
+    &_view_normal {
+        --_-g-number-input-numeric-arrows-border-color: var(--g-color-line-generic);
+        --_--border-color-hover: var(--g-color-line-generic-hover);
+        --_--border-color-active: var(--g-color-line-generic-active);
+
+        &#{$block}_state_error {
+            --_-g-number-input-numeric-arrows-border-color: var(--g-color-line-danger);
+        }
+    }
+
+    &_view_clear {
+        --_-g-number-input-numeric-arrows-border-color: transparent;
+    }
+
+    &__numeric-arrows {
```

```suggestion
    &__arrows {
```

Let's be compact here